Even though linting does work, Inoticed it is throwing an error when running linting command. Need to investigate more 
```bash
Error: Failed to load parser '@typescript-eslint/parser' declared in '.eslintrc.js » @flexn/eslint-config': Cannot find module '@typescript-eslint/parser'
Require stack:
- /Users/marijus/Documents/lint/packages/eslint-config/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.resolve (internal/modules/cjs/helpers.js:94:19)
    at Object.resolve (/Users/marijus/Documents/projects/prj-036-esc/node_modules/@eslint/eslintrc/lib/shared/relative-module-resolver.js:28:50)
    at ConfigArrayFactory._loadParser (/Users/marijus/Documents/projects/prj-036-esc/node_modules/@eslint/eslintrc/lib/config-array-factory.js:929:45)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/marijus/Documents/projects/prj-036-esc/node_modules/@eslint/eslintrc/lib/config-array-factory.js:724:43)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/Users/marijus/Documents/projects/prj-036-esc/node_modules/@eslint/eslintrc/lib/config-array-factory.js:665:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/Users/marijus/Documents/projects/prj-036-esc/node_modules/@eslint/eslintrc/lib/config-array-factory.js:720:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
```

--- a/lerna.json
+++ b/lerna.json
@@ -1,5 +1,5 @@
 {
-    "version": "0.1.4",
+    "version": "0.1.5-alpha.0",
     "packages": [
         "packages/*"
     ],

--- a/packages/eslint-config/package.json
+++ b/packages/eslint-config/package.json
@@ -1,6 +1,6 @@
 {
     "name": "@flexn/eslint-config",
-    "version": "0.1.4",
+    "version": "0.1.5-alpha.0",
     "description": "Flexn ESLint config",
     "main": "index.js",
     "scripts": {},

